### DISCUSSION TITLE
Build Trivy in Docker

### DISCUSSION THREAD

**Discussion by NikolayPopovDev** [aquasecurity/trivy#6322] [Development]
I want to build Trivy with my Dockerfile, but it didn't work.
There is my Dockerfile:

```
FROM golang:alpine as trivy
RUN apk add --no-cache git
WORKDIR /usr/local/bin/trivy
COPY . .
RUN go get -d -v ./...
RUN go build -v ./...
```

I ran build image and have errors:

```
    github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/php/composer: cannot find module providing package github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/php/composer
    go: module github.com/golang/protobuf is deprecated: Use the "google.golang.org/protobuf" module instead.
```

Command `go get -d` also didn't work locally with the same errors. I'm using MacOS Monterey and Go 1.22
Can you help me?

**Comment by DmitriyLewen** [comment]
Hello @NikolayPopovDev 

Can you try `go build -o trivy ./cmd/trivy/main.go `?

Regards, Dmitriy

**Reply by NikolayPopovDev** [reply]
Hey @DmitriyLewen 

Thanks for reply. I found where is mistake

**Comment by NikolayPopovDev** [comment]
The error was because in my .dockerignore file was `**/compose*` line
